src/global_styling/variables/_states.scss

```diff
@@ -20,8 +20,8 @@ $ouiFocusRingSizeLarge: $ouiSizeXS !default;
 $ouiFocusRingSize: $ouiFocusRingSizeLarge * .75 !default;
 
 // Transparency
-$ouiFocusTransparency: lightOrDarkTheme(.1, .3) !default;
-$ouiFocusBackgroundColor: tintOrShade($ouiColorPrimary, ((1 - $ouiFocusTransparency) * 100%), ((1 - $ouiFocusTransparency) * 100%)) !default;
+$ouiFocusTransparency: lightOrDarkTheme(90%, 70%) !default;
+$ouiFocusBackgroundColor: tintOrShade($ouiColorPrimary, $ouiFocusTransparency, $ouiFocusTransparency) !default;
```

Comment:
The simplification reveals that this definition itself is a bit suspect. If you search the OUI stylesheets for `tintOrShade` usage, you'll see that it's most common to specify more `tint` than `shade` (or, occasionally, the reverse). That makes sense from a color perception perspective.

There are only 2 other examples I found where the two percentages are the same:

https://github.com/opensearch-project/oui/blob/fe979c3662afc5e4b620f05c838d4b98beb7573e/src/global_styling/variables/_buttons.scss#L16

https://github.com/opensearch-project/oui/blob/fe979c3662afc5e4b620f05c838d4b98beb7573e/src/components/control_bar/_control_bar.scss#L177

So I don't think we should be afraid to further update this definition to be more in tune with other derived colors.

Reply:
I think the point of tint or shade is more to specifically get the tint vs the shade functionality depending on dark or light colors. The actual color switching is done by the `$ouiFocusTransparency` variable, so we will only ever get tint 90% on light mode and shade 70% on dark mode. It's just separated into 2 separate variables